src/app/pages/network/configuration/configuration.component.ts

```diff
@@ -31,19 +31,19 @@ export class ConfigurationComponent {
       type : 'input',
       name : 'gc_hostname',
       placeholder : T('Hostname'),
-      tooltip : T('System host name.'),
+      tooltip : T('Enter the system hostname.'),
     },
     {
       type : 'input',
       name : 'gc_domain',
       placeholder : T('Domain'),
-      tooltip : T('System domain name.'),
+      tooltip : T('Enter the system domain name.'),
     },
     {
       type : 'textarea',
       name : 'gc_domains',
       placeholder: T('Additional Domains'),
-      tooltip : T('Add other domains to search, separated by spaces.\
+      tooltip : T('Enter additional space-delimited domains to search.\
```

Comment:
As above, leave out the "Enter" that makes this imperative.

Reply:
done